source/js/header.js

```diff
@@ -82,6 +96,19 @@
       document.getElementById('country-badge').className = "flag-icon flag-icon-squared flag-icon-" + countryCode;
     }
   }
+  
+  function getQueryVariable(variable) {
+      var query = window.location.search.substring(1);
+      var vars = query.split('&');
```

Comment:
nit: just make this vars = window.location.seach.substring(1).split('&')? you don't seem to be using query anywhere else.